Protos/V1/partitions_common.proto

```diff
@@ -115,3 +117,82 @@ message GetPartitionRequest {
 message GetPartitionResponse {
   PartitionRaw partition = 1; /** The raw partition. */
 }
+
+/**
+ * Request to update multiple partitions.
+ *
+ * Cannot update more than 30 partitions
+ */
+message UpdatePartitionsRequest {
+  /**
+   * Represents the data of the partitions that will be updated.
+   *
+   * ID is mandatory.
+   * -1 means that value will not be updated
```

Comment:
I would just say negative instead of specifically -1.
The problem is that in protobuf, if a value is missing, it is set to 0...

Reply:
Ok.
But 0 is a valid number for our use case...